flight/drone/drone_controller.py

```diff
@@ -16,7 +19,7 @@
 from ..tasks.takeoff_task import TakeoffTask
 from ..utils.priority_queue import PriorityQueue
 from ..utils.timer import Timer
-from ... import flightconfig as f
+
```

Comment:
drone_controller.py needs to use the flight config constants for things like safety checks. I don't really understand these relative imports and why flightconfig is the only one that is causing issues.
```suggestion
import flightconfig as fc
```

Reply:
My mistake, I don't believe this was an intentional change.